packages/infrastructure/sqldb/main.tf

```diff
@@ -0,0 +1,32 @@
+terraform {
```

Comment:
Better to call this folder for `api` as it sets up resources needed for the `api` package. Easier to navigate if the infrastructure folders somewhat matches the packages.